tests/helpers.py

```diff
@@ -72,17 +75,20 @@ def run_performance_cli(timing_regex, timing_database):
     run_cli(performance, args)
 
 
-def run_tolerance_cli(stats_file_name, tolerance_file_name):
+def run_tolerance_cli(
+    stats_file_name, tolerance_file_name, member_type=None, member_num=10
+):
     args = [
         "--stats-file-name",
         stats_file_name,
         "--tolerance-file-name",
         tolerance_file_name,
         "--member-num",
-        "10",
-        "--member-type",
-        "dp",
+        str(member_num),
     ]
+    if member_type:
```

Comment:
Maybe pedantic. But if I understood correctly, I would prefer:
```suggestion
    if member_type is not None:
```